.github/CODEOWNERS

```diff
@@ -322,7 +315,6 @@ azure-pipelines.yml @coq/ci-maintainers
 ########## Developer tools ##########
 
 /dev/tools/backport-pr.sh     @Zimmi48
-# Secondary maintainer @maximedenes
```

Comment:
@vbgl Do you want to be the secondary maintainer given that you are the current RM?

Reply:
Sure.